src/components/form/Menu.vue

```diff
@@ -0,0 +1,52 @@
+<script setup>
+import { reactive, computed } from 'vue'
+
+
+// site name validation
+const state = reactive({
+  siteName: 'Choisis'
+})
+
+
+// submitted status
+const myComputedProp = computed(() => {
+  return state.siteName + ' Ton Menu'
+})
+
+</script>
+
+<template>
+    <aside>
+    <div>
+        <ul>
+            <li>
+                <a href="/">{{myComputedProp}}</a>
+            </li>
+            <li>
+                <a href="new1">Test</a>
+            </li>
+            <li>
+                <a href="new2">tes2</a>
+            </li>
+            <li>
+                <a href="new3">test3</a>
+            </li>
+            <li>
+                <a href="ToDoList">ToDoList</a>
+            </li>
+        </ul>
+    </div>
+</aside>
+</template>
+<style scoped>
+aside {
+    position: fixed;
```

Comment:
position: fixed is not going to make it for this kind of solution, you need to use display: flex
Check out https://css-tricks.com/snippets/css/a-guide-to-flexbox/ for tips on how to use it.
Come see me if you have issues.